src/test/java/teammates/test/cases/datatransfer/FeedbackSessionAttributesTest.java

```diff
@@ -26,8 +26,10 @@
 
     @BeforeClass
     public void classSetup() {
-        startTime = TimeHelper.combineDateTime("09/05/2016", "1000");
```

Comment:
Note that this `"1000"` is a mistake, since `combineDateTime` is supposed to take in the hour only, not the minute